src/systems/ec/implementations/react/templates/pages/person/src/PersonPage.jsx

```diff
@@ -0,0 +1,907 @@
+                    title: 'Milestone',
+                  },
+                  {
+                    id: 7,
+                    label: '...',
+                    title: 'Milestone',
+                  },
+                  {
+                    id: 8,
+                    label: '...',
+                    title: 'Milestone',
+                  },
+                  {
+                    id: 9,
+                    label: '...',
+                    title: 'Milestone',
+                  },
+                  {
+                    id: 10,
+                    label: '...',
+                    title: 'Milestone',
+                  },
+                  {
+                    id: 11,
+                    label: '...',
+                    title: 'Milestone',
+                  },
+                  {
+                    id: 12,
+                    label: '1984',
+                    title: "Master's degree in History, Vilnius University",
+                  },
+                  {
+                    id: 13,
+                    label: '1975',
+                    title:
+                      "Medical doctor's degree, Kaunas Institute of Medicine",
+                  },
+                ]}
+              />
+
+              <FileDownload
+                data-ecl-auto-init="FileDownload"
+                className="ecl-u-mt-none ecl-u-mb-xl ecl-u-mb-lg-3xl"
+                title="Vytenis Andriukaitis' CV"
+                language="English"
+                meta="(43.8 KB - PDF)"
+                icon={{
+                  shape: 'general--copy',
+                  size: '2xl',
+                }}
+                download={{
+                  label: 'Download',
+                  href: '/example',
+                }}
+              />
+
+              <h2
+                className="ecl-u-type-heading-2 ecl-u-type-color-black ecl-u-mt-none ecl-u-mb-l ecl-u-mb-lg-l"
+                id="responsibilities"
+              >
+                Responsibilities
+              </h2>
+
+              <UnorderedList className="ecl-u-mt-none ecl-u-mb-xl ecl-u-mb-lg-l">
+                <UnorderedListItem>
+                  modernising and simplifying EU food safety policy while
+                  keeping the current high level of safety and ensuring existing
+                  policies have maximum effect.
+                </UnorderedListItem>
+                <UnorderedListItem>
+                  ensuring the Commission is ready in supporting the EU&apos;s
+                  capacity to deal with crisis situations in food safety or
+                  pandemics
+                </UnorderedListItem>
+                <UnorderedListItem>
+                  reviewing the laws that oblige the Commission to authorise
+                  genetically modified organisms (GMOs), even when a majority of
+                  national governments oppose them.
+                </UnorderedListItem>
+              </UnorderedList>
+
```

Comment:
In my personal opinion: content should not be as important for the demo. I do add the different types of content like links, un-ordered lists, etc. but I just do not plan to input all the content from the image, because it takes unnecessary time to copy paste. If it is important, I will need the actual text to copy-pate.